lib/Table/ColorStyleLegend.ts

```diff
@@ -37,8 +37,27 @@ export class ColorStyleLegend extends LoadableStratum(LegendTraits) {
     return this.catalogItem.activeTableStyle;
   }
 
+  // Keep these here until we deprecate LegendTraits in TableColorStyleTraits
+  @computed get oldLegendTraits() {
+    return this.tableStyle.colorTraits.legend;
+  }
+
+  @computed get url() {
+    return this.oldLegendTraits.url;
+  }
+  @computed get imageScaling() {
+    return this.oldLegendTraits.imageScaling;
+  }
+  @computed get urlMimeType() {
+    return this.oldLegendTraits.urlMimeType;
+  }
+  @computed get backgroundColor() {
+    return this.oldLegendTraits.backgroundColor;
+  }
+
```

Comment:
Can you link to an issue about deprecating LegendTraits in TableColorStyleTraits?

Reply:
Done